en/engine/event/event-api.md

```diff
@@ -0,0 +1,61 @@
+# Global and Node Touch and Mouse Events API
+
+## Mouse Event API
+
+| Function Name | Return value type | Meaning                                                                                 |
+|---------------|-------------------|-----------------------------------------------------------------------------------------|
+| getScrollY    | Number            | get the scrolling distance of the mouse wheel on the y-axis, valid only when scrolling. |
+| getButton       | Number | cc.Event.EventMouse.BUTTON_LEFT or cc.Event.EventMouse.BUTTON_RIGHT or cc.Event.EventMouse.BUTTON_MIDDLE.                     ｜
+
+### Global Mouse Events API
+
+| Function Name       | Return value type | Meaning                                                                                             |
+|---------------------|-------------------|-----------------------------------------------------------------------------------------------------|
+| getLocation         | Vec2              | get the mouse position, which contains the x and y properties.                                      |
+| getLocationX        | Number            | get the mouse position on x-axis.                                                                   |
+| getLocationY        | Number            | get the mouse position on y-axis.                                                                   |
+| getPreviousLocation | Vec2              | get the position of the last triggered mouse event, which contains the x and y properties.          |
+| getDelta            | Vec2              | get the distance object the mouse moves since last event, which includes x and y properties.        |
```

Comment:
```suggestion
| getDelta            | Vec2              | get the distance the mouse has moved relative to the lower-left corner since the last event, which contains the x and y properties.        |
```